### ISSUE TITLE
direction field in EEP

### ISSUE THREAD

**Issue by romor** [kipe/enocean#6]
The interpretation of some EEPs is not sufficiently specified with `RORG`, `FUNC` and `TYPE` but needs an additional direction flag (see e.g. A5-20-01).

How could this be covered best?
By an optional `direction`attribute of the `profile` xml tag?
Or an optional `direction` tag as a child of the `profile` xml tag?

**Comment by kipe** [comment]
Hmm, this is quite interesting...
I'd go for multiple `data` -tags in the `profile`, which have an optional `direction` -tag. Something like this:
```xml
  <telegram rorg="0xD5" type="1BS">
    <profiles description="Contacts and switches" func="0x00">
      <profile type="0x01">
        <data direction="1">
          <enum shortcut="CO" description="Contact" offset="7" size="1">
            <item description="open" value="0" />
            <item description="closed" value="1" />
          </enum>
        </data>
        <data direction="2">
          <enum shortcut="CO" description="Contact" offset="7" size="1">
            <item description="open" value="0" />
            <item description="closed" value="1" />
          </enum>
        </data>
      </profile>
    </profiles>
  </telegram>
```

This way, in case there's only one `data` declaration, we can use it both for transmit and receive. In case there are multiple, we can choose based on the direction. 

Also, the incoming message parsing should always be first in the XML, for consistency. I haven't checked the documentation more thoroughly, but I assume it's this way in there also. If not, we still have the `direction` attribute to follow, no matter what.

If the directions seem to be consistent in the documentation, they should also be declared in the `enocean.protocol.constants`.

But anyways, do experiments and see which seems to be the best way. For this, I'll require thorough testing, as there's a big possibility for breaking something :smile:

**Comment by kipe** [comment]
Does PR #12 this, if (and when) merged?

**Comment by romor** [comment]
No, PR #12 does not address this issue.

**Comment by kipe** [comment]
Closed with PR #18
